fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -4143,7 +4144,11 @@ struct StartFullRestoreTaskFunc : RestoreTaskFuncBase {
 			}
 		}
 
-		if (!CLIENT_KNOBS->RESTORE_IGNORE_LOG_FILES) {
+		tr->reset();
+		tr->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+		tr->setOption(FDBTransactionOptions::LOCK_AWARE);
+		bool inconsistentSnapshotOnly = wait(restore.inconsistentSnapshotOnly().getD(tr));
```

Comment:
For clarity, let's pass `false` as the default arg for `getD()`.  Note that this is the third argument, so you must do `getD(tr, false, false)`.  

And yes this was an awkward choice of argument order on my part, I want to very carefully fix that some day and maybe rename this to `getOrDefault()` to align better with the `Optional` interface.